xbmc/windowing/gbm/drm/DRMUtils.cpp

```diff
@@ -73,13 +73,13 @@ drm_fb * CDRMUtils::DrmFbGetFromBo(struct gbm_bo *bo)
   fb->bo = bo;
   fb->format = m_gui_plane->GetFormat();
 
-  uint32_t width,
-           height,
-           handles[4] = {0},
-           strides[4] = {0},
-           offsets[4] = {0};
+  uint32_t width;
```

Comment:
These can be declared where they're first used, to avoid temporarily having uninitialized memory lying around.

Reply:
In this commit I only wanted to replace `{0}` with more generic `{}`

Reply:
If you don't want to move lines then I would zero-initialize `uint32_t width = 0;` to avoid uninitialized memory. You're trading theoretical performance for theoretical safety, and I would argue in favor of safety.

Reply:
I have removed the changes here (it is left as is). The reason is that the commit tries to make use of {} instead of {0} to initialize structures (with different types) but in this case they are integer arrays where it is perfectly logical and correct to use {0}. Maybe there are other reasons to modify / improve this code but they have nothing to do with this commit.

Reply:
This weird operator overloading has been a criticism of C++ before :slightly_smiling_face: